lib/resources/dstu2/schedule.yaml

```diff
@@ -0,0 +1,22 @@
+---
+name: Schedule
+field_name_base_url: http://hl7.org/fhir/DSTU2/schedule-definitions.html#Schedule
+fields:
+- name: type
+  required: 'No'
+  cardinality: 0..1
+  type: CodeableConcept
+  description: The schedule type can be used for the categorization of healthcare services or other appointment types.
+  example: |
+             {"type": {
+                "coding": [
+                  {"system": "http://snomed.info/sct",
+                   "code": "394581000",
+                   "display": "Community medicine",
+                   "userSelected": false}],
+                "text": "Same Day"}
+  binding:
+    description:  Additional details about where the content was created (e.g. clinical specialty).
+    terminology:
+    - display: Practice Setting Code Value Set
+      system: http://hl7.org/fhir/ValueSet/c80-practice-codes
```

Comment:
Use the newer system and info_link fields to separate the documentation of the value set (which should link to a DSTU 2 specific link) and the actual system that will be in the response (in this case, SNOMED)